### PR TITLE
day_03

### PR DESCRIPTION
**Pull request by berataras**

### CODE REVIEW

File: 03_Booleans_Operatörler_Tarih/README.md [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+
+```js
+let firstName = "Prototurk";
+let country = "Türkiye";
+```
+
+Assignment Operators (Atama Operatörleri)
+
+![Assignment operators](../images/assignment_operators.png)
+
```
Comment (fahridev) [inline 03_Booleans_Operatörler_Tarih/README.md:68]: [] => empty
(https://github.com/Asabeneh/30-Days-Of-JavaScript/raw/master/images/assignment_operators.png)

File: 03_Booleans_Operatörler_Tarih/README.md [hunk 2]
```diff
@@ -0,0 +1,444 @@
...
+console.log(
+  `The boiling point of water is ${boilingPoint} oC.\nHuman body temperature is ${bodyTemp} oC.\nThe gravity of earth is ${gravity} m / s2.`
+);
+```
+
+### Comparison Operators (Karşılaştırma Operatörleri)
+
+Programlamada değerleri karşılaştırırz ve iki değeri karşılaştırmak için karşılaştırma operatörlerini kullanırız. Bir değerin diğer değere eşit veya büyük olup olmadığını kontrol ederiz.
+
+![Comparison Operators](../images/comparison_operators.png)
```
Comment (fahridev) [inline 03_Booleans_Operatörler_Tarih/README.md:121]: [] => empty
(https://github.com/Asabeneh/30-Days-Of-JavaScript/raw/master/images/comparison_operators.png)

File: 03_Booleans_Operatörler_Tarih/README.md [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+Tüm metodlar bunlarla sınırlı değil. Ayrı bir bölümde bu metodları derinlemesine işleyeceğiz.
+
+## Date Object (Tarih Nesnesi)
+
+Belirli bir faaliyetin veya olayın zamanını bilmek isteriz. Javascript'te geçerli zaman ve tarih, Date Object kullanılarak oluşturulur.
+
+Oluşturduğumuz Date objesi bize bir çok zaman ve tarih metodu sağlar. Kullandığımız metodlarla tarih ve zaman bilgilerini alabiliriz. Genel de bu metodlar "almak" anlamına gelen _get_ ile başlar.
+_getFullYear(), getMonth(), getDate(), getDay(), getHours(), getMinutes, getSeconds(), getMilliseconds(), getTime(), getDay()_
+
+![Date time Object](../images/date_time_object.png)
```
Comment (fahridev) [inline 03_Booleans_Operatörler_Tarih/README.md:330]: [] => empty
(https://github.com/Asabeneh/30-Days-Of-JavaScript/raw/master/images/date_time_object.png)

Comment (berataras) [reply 03_Booleans_Operatörler_Tarih/README.md:330]: eyvallah kardo ya çok üşeniyordum bunu yapmaya hahaha

**Pull request closed without merging**